package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gen3/guppy",
-  "version": "0.15.7",
+  "version": "0.15.8",
```

Comment:
I think we should bump a minor version for it since this PR adds new features to the text agg queries